## ComfyUI-Manager: installing dependencies done.
[2024-11-23 12:22:07.531] ** ComfyUI startup time: 2024-11-23 12:22:07.531556
[2024-11-23 12:22:07.531] ** Platform: Darwin
[2024-11-23 12:22:07.531] ** Python version: 3.11.10 (main, Sep  7 2024, 01:03:31) [Clang 15.0.0 (clang-1500.3.9.4)]
[2024-11-23 12:22:07.531] ** Python executable: /Users/aethio/.kipp/comfy-venv/bin/python
[2024-11-23 12:22:07.531] ** ComfyUI Path: /Users/aethio/.kipp/ComfyUI
[2024-11-23 12:22:07.531] ** Log path: /Users/aethio/Projects/piquant/piquant-refresh/comfyui.log
[2024-11-23 12:22:07.535] 
Prestartup times for custom nodes:
[2024-11-23 12:22:07.535]    0.4 seconds: /Users/aethio/.kipp/ComfyUI/custom_nodes/ComfyUI-Manager
[2024-11-23 12:22:07.535] 
Total VRAM 32768 MB, total RAM 32768 MB
[2024-11-23 12:22:08.313] pytorch version: 2.5.1
[2024-11-23 12:22:08.313] Set vram state to: SHARED
[2024-11-23 12:22:08.313] Device: mps
[2024-11-23 12:22:08.771] Using sub quadratic optimization for cross attention, if you have memory or speed issues try using: --use-split-cross-attention
[2024-11-23 12:22:09.236] [Prompt Server] web root: /Users/aethio/.kipp/ComfyUI/web
[2024-11-23 12:22:09.424] ### Loading: ComfyUI-Manager (V2.51.9)
[2024-11-23 12:22:09.493] ### ComfyUI Revision: 2816 [dd5b57e3] | Released on '2024-11-08'
[2024-11-23 12:22:09.495] 
Import times for custom nodes:
[2024-11-23 12:22:09.495]    0.0 seconds: /Users/aethio/.kipp/ComfyUI/custom_nodes/websocket_image_save.py
[2024-11-23 12:22:09.495]    0.1 seconds: /Users/aethio/.kipp/ComfyUI/custom_nodes/ComfyUI-Manager
[2024-11-23 12:22:09.495] 
[2024-11-23 12:22:09.499] Starting server

[2024-11-23 12:22:09.500] Traceback (most recent call last):
[2024-11-23 12:22:09.500]   File "/Users/aethio/.kipp/ComfyUI/main.py", line 261, in <module>
[2024-11-23 12:22:09.500]     loop.run_until_complete(run(server, address=args.listen, port=args.port, verbose=not args.dont_print_server, call_on_start=call_on_start))
[2024-11-23 12:22:09.500]   File "/opt/homebrew/Cellar/python@3.11/3.11.10/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
[2024-11-23 12:22:09.500]     return future.result()
[2024-11-23 12:22:09.500]            ^^^^^^^^^^^^^^^
[2024-11-23 12:22:09.500]   File "/Users/aethio/.kipp/ComfyUI/main.py", line 166, in run
[2024-11-23 12:22:09.501]     await asyncio.gather(server.start_multi_address(addresses, call_on_start), server.publish_loop())
[2024-11-23 12:22:09.501]   File "/Users/aethio/.kipp/ComfyUI/server.py", line 840, in start_multi_address
[2024-11-23 12:22:09.501]     await site.start()
[2024-11-23 12:22:09.501]   File "/Users/aethio/.kipp/comfy-venv/lib/python3.11/site-packages/aiohttp/web_runner.py", line 119, in start
[2024-11-23 12:22:09.501]     self._server = await loop.create_server(
[2024-11-23 12:22:09.501]                    ^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-11-23 12:22:09.501]   File "/opt/homebrew/Cellar/python@3.11/3.11.10/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 1536, in create_server
[2024-11-23 12:22:09.501]     raise OSError(err.errno, msg) from None
[2024-11-23 12:22:09.501] OSError: [Errno 48] error while attempting to bind on address ('127.0.0.1', 8188): address already in use
[2024-11-23 12:22:09.604] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[2024-11-23 12:22:09.604] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[2024-11-23 12:22:09.736] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
[2024-11-23 12:22:09.774] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[2024-11-23 12:22:09.790] [ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
